torch/onnx/__init__.py

```diff
@@ -13,13 +13,17 @@
 
 
 class ExportTypes:
-    PROTOBUF_FILE = 1
-    ZIP_ARCHIVE = 2
-    COMPRESSED_ZIP_ARCHIVE = 3
-    DIRECTORY = 4
+    r""""Specifies how the ONNX graph is stored"""
```

Comment:
Nit: Everywhere `ONNX graph` -> `ONNX model`

Technically we export models, not graphs. The model contains a graph and other metadata.

I personally would also omit `ONNX` since I think it's pretty obvious from the context, but up to you.